packages/app/src/components/nominationInfo/healthProviderSearch/index.js

```diff
@@ -49,19 +49,19 @@ const SearchHealthProvider = () => {
   return (
   <>
     <div className="search-bar-wrapper">
-      <section className="row">
+      {/* <section className="row"> */}
```

Comment:
Was commenting out this line intentional? If so, we should delete the line instead.

Reply:
Removed comments.